### PR TITLE
implements HTTP GET nodes for the given cluster.

### PR DESCRIPTION
**Pull request by p0lyn0mial**
**What this PR does / why we need it**: implements HTTP GET nodes for the given cluster.

**Fixes**: #1371

**Special notes for your reviewer**: #1371

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: api/pkg/handler/node_v1_test.go [hunk 1]
```diff
@@ -9,14 +9,279 @@ import (
 	"testing"
 
 	apiv1 "github.com/kubermatic/kubermatic/api/pkg/api/v1"
+	apiv2 "github.com/kubermatic/kubermatic/api/pkg/api/v2"
 	kubermaticv1 "github.com/kubermatic/kubermatic/api/pkg/crd/kubermatic/v1"
 	"github.com/kubermatic/machine-controller/pkg/machines/v1alpha1"
 
+	"io/ioutil"
```
Comment (mrIncompetent) [inline api/pkg/handler/node_v1_test.go]: grouping

**Review by mrIncompetent — DISMISSED** [review]

File: api/pkg/handler/node_v1_test.go (hunk 1 above)
Comment (p0lyn0mial) [reply api/pkg/handler/node_v1_test.go]: :(

Comment (p0lyn0mial) [reply api/pkg/handler/node_v1_test.go]: fixed in https://github.com/kubermatic/kubermatic/pull/1580

**Comment by p0lyn0mial** [comment]
@mrIncompetent PTAL

**Review by mrIncompetent — APPROVED** [review]

**Pull request merged**